doc/articles/faq.md

```diff
@@ -1,3 +1,138 @@
+Uno Platform was created by Team Nventive, add more stuff here
+
+-What makes UNO Platform unique?
+
+-Where does the name Uno Platform come from? 
+
+-Is Uno Platform open source?
+
+ **Getting started**
+**How can I try Uno platform?**
+Playground, quick start
+
+**How experienced do I need to be to use Uno Platform?**
+
+
+**What is the playground?**
+
+**Where can I get support?**
+
+**How can I get involved?**
+There are lots of ways to contribute to the Uno Platform and we appreciate the help from the community. You can provide feedback, report bugs, give suggestions, contribute code, and participate in the platform discussions.
+
+**How can I report a bug?**
+-If you think you've found a bug, please log a new issue in the Uno Platform GitHub issue tracker. When filing issues, please use our bug filing template. The best way to get your bug fixed is to be as detailed as you can be about the problem. Providing a minimal project with steps to reproduce the problem is ideal. Here are questions you can answer before you file a bug to make sure you're not missing any important information.
+-Did you read the documentation?
+-Did you include the snippet of broken code in the issue?
+What are the EXACT steps to reproduce this problem?
+What specific version or build are you using?
+What operating system are you using?
+GitHub supports markdown, so when filing bugs make sure you check the formatting before clicking submit.
+
+**What is UWP?** 
+Universal Windows Platform (UWP) is an API created by Microsoft and first introduced in Windows 10. The purpose of this platform is to help develop universal apps that run on Windows 10, Windows 10 Mobile, Xbox One and HoloLens without the need to be re-written for each. It supports Windows app development using C++, C#, VB.NET, and XAML. The API is implemented in C++, and supported in C++, VB.NET, C#, F# and JavaScript. Designed as an extension to the Windows Runtime platform first introduced in Windows Server 2012 and Windows 8, UWP allows developers to create apps that will potentially run on multiple types of devices.
+
+Visit Microsoft's documentation for a primer on UWP : https://docs.microsoft.com/en-us/windows/uwp/get-started/
+
+**Technology**
+**How is Uno platform different from Xamarin forms?**
+
+**How does my Uno platform code run on Android?**
+
+**How does my Uno platform code run on iOS?**
+
+**How does my Uno platform code run on the web?**
+
+**Is WebAssembly supported in all browers?**
+
+**Where can I deploy my Uno Platform app?**
+
+**Does Uno.UI support macOS?**
+Not yet. We're still waiting for Apple's Marzipan cross macOS-iOS APIs to become available. We’ll see from there. 
+
+**Does Uno.UI support Linux ?**
+No, not at the moment. Our assumption for now is that the WebAssembly part of Uno.UI will be efficient enough to be a viable alternative. If you find that native support still is a viable scenario, please open a GitHub issue.
+
+## Does Uno.UI support wpf CustomControls 
```

Comment:
'Does Uno.UI support CustomControls?' (The 'wpf' is confusing, Uno isn't compatible with WPF)

Reply:
Yeah WPF or Silverlight can be reused but they'll need some modifications. and if we refer to WPF we should use capital letters :)